### ISSUE TITLE
Windows phone, touch highlights wrong point with page zoom

### ISSUE THREAD

**Issue by TorsteinHonsi** [highcharts/highcharts#4924]
1. Open http://jsfiddle.net/highcharts/tgchL9k1/show
2. Pinch-zoom
3. Touch any point. The tooltip displays the wrong point, but in the right position – e.g. if I tap on June, April label is selected and shown

Split from #4886

**Comment by TorsteinHonsi** [comment]
_Internal note:_

What I've found so far is that in the `Pointer.normalize` function, when we try to determine `chartX`, the result differs if we have touch-scrolled away from the left edge of the page. The `offset` function should return a left value of 0, but it adds the touch-scroll offset even though the `e.pageX` property doesn't include it. It is probably caused by the `window.pageXOffset` being added on Windows Mobile, even though it shouldn't because we're only moving the viewport around.

**Comment by sebastianbochan** [comment]
+1

**Comment by sebastianbochan** [comment]
+1

**Comment by Wanderduene** [comment]
Hello,

we are using Highcharts in our software and got a bug report about the behavior of our charts on the Microsoft Surface tablet, and I think the bug is identical to this one.
The issue is simple to reproduce, also with the demo charts on your homepage:

1. Open: http://www.highcharts.com/demo/line-basic
2. Zoom in with the pinch gesture
3. Tap on a point
Result: The wrong point get's selected.

Do you think this is something that can be fixed, or is it an issue with the specific device?


_System:_
Surface Pro 3
Windows 10 Enterprise
64 Bit

_Browser:_
Microsoft Edge 25.10586.0.0
Microsoft EdgeHTML 13.10586

_Highcharts Info:_ 
Version 4.2.5

**Comment by TorsteinHonsi** [comment]
I can confirm this, reproduced on Windows Phone via BrowserStack.

It seems to be the same issue as [this unresolved Stack Overflow question](http://stackoverflow.com/questions/36758507/zoom-offsettop-in-edge).

I haven't been able to find a property that describes the current page zoom, so no workaround yet. If we find it, the appropriate place to add it would be in the `Pointer.normalize` function, where we can do something similar to https://github.com/highcharts/highcharts/commit/68084085b5190c78152c4c3c4fd0fa1440899b69.

**Comment by Wanderduene** [comment]
Okay, thank you for the information!

**Comment by guillaumeprevot** [comment]
Hello, I had the same issue on Edge, with Windows 10 Pro on a laptop with touchscreen.
The **following fix worked for me with HighChart 5.0.12**.
Add this 3 lines in Pointer.normalize, at line 13095 of highcharts.src.js, just before the return statement :

```javascript
// Fix IE/Edge offset bug due to pinch zoom
if (window.scrollX)
    chartX += window.scrollX;
```

Oddly, there's no need to deal with window.scrollY.
